static/styles/portico/portico.css

```diff
@@ -246,7 +253,7 @@ html {
 
         a {
             &.highlighted {
-                background-color: hsl(152, 40%, 42%);
+                background-color: $text2;
```

Comment:
To test this PR change this to $text1 or $text3.
Also do change the blue one given above.